malleefowl/persist.py

```diff
@@ -8,38 +9,67 @@
 LOGGER = logging.getLogger("PYWPS")
 
 
-def resolve(location, place_holders, f):
+def resolve(location, f, defaults=None):
+    place_holders = re.findall('\{(.*?)\}', location)
+    if not place_holders:
+        return location
```

Comment:
In that case we miss the opportunity to add defaults value to the dataset metadata, right?